README.md

```diff
@@ -4,6 +4,10 @@ Status: This is **ALPHA** software.
 
 [![Build status](https://badge.buildkite.com/0bcfe58b6f5741aacb09b12485969ba7a1205955a45b53e854.svg)](https://buildkite.com/bazel/python-rules-python-postsubmit)
 
+## Recent updates
```

Comment:
I'm often a little unclear on whether README.md  refers to the most recent release of the repository (and thus should only be updated alongside cutting a new release) or whether it affects state at HEAD. I leave it up to you -- just wanted to make sure you've thought about its role in the release process of rules_python :)

Reply:
Good point. I don't feel too bad about it being HEAD given that this repo doesn't even have any tags, but I don't know what a general solution would be. Aside from not iterating so quickly on the API. :)